appinsights/client.go

```diff
@@ -26,8 +27,11 @@ type telemetryClient struct {
 }
 
 func NewTelemetryClient(iKey string) TelemetryClient {
-	config := NewTelemetryConfiguration(iKey)
-	channel := NewInMemoryChannel(config.EndpointUrl)
+	return NewTelemetryClientFromConfig(NewTelemetryConfiguration(iKey))
+}
+
+func NewTelemetryClientFromConfig(config *TelemetryConfiguration) TelemetryClient {
```

Comment:
It would be great to come up with common names for telemetry client generation functions for all of our SDKs. Right now Node uses `getClient(ikey)` which is endlessly confusing naming